src/styles/annotator/notebook.scss

```diff
@@ -2,7 +2,6 @@
 @use '../mixins/molecules';
 
 .notebook-outer {
-  // TODO: CSS namespace conflicts?
```

Comment:
I believe this comment referred to conflicts with host page styles. This is no longer something we need to worry about with annotator CSS, assuming that the styles are applied to an element in shadow DOM. For the small percentage of browsers that can load Hypothesis but don't support shadow DOM (mainly legacy Edge and possibly a few users on old versions of Firefox), we just accept a small risk of style conflicts.